daemon/src/lib.rs

```diff
@@ -180,13 +180,17 @@ where
             .checked_mul(2)
             .expect("not to overflow");
 
-        tasks.add(connection_actor_ctx.run(connection::Actor::new(
-            maker_online_status_feed_sender,
-            &cfd_actor_addr,
-            identity.identity_sk,
-            taker_heartbeat_timeout,
-            connect_timeout,
-        )));
+        tasks.add(
+            connection_actor_ctx
+                .with_handler_timeout(Duration::from_secs(120))
```

Comment:
Given 120 is arbitrary, maybe it's worth mentioning this in the code or in the commit message that this is arbitrary. Otherwise we will wonder in the future why this number was chosen.

Reply:
agreed. it might be worth extracting as a const so that it's easily found at the later point! (then a comment would have a nice spot too).

Reply:
I agree, will do.

Reply:
as follow up please :)